.github/CODEOWNERS

```diff
@@ -165,6 +165,7 @@ pkg/winperfcounters/                                                @open-teleme
 
 processor/attributesprocessor/                                      @open-telemetry/collector-contrib-approvers @boostchicken
 processor/cumulativetodeltaprocessor/                               @open-telemetry/collector-contrib-approvers @TylerHelmuth
+processor/coralogixprocessor/                                       @open-telemetry/collector-contrib-approvers @galrose @crobert-1 @eyalatz @roycald245
```

Comment:
This broke a CI in mainline: https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/10317079346/job/28560709557#step:16:11

> codeowners are not members: eyalatz, roycald245

Reply:
how do we fix this? is this because we are missing in `cmd/githubgen/allowlist.txt`?